net/ip_test.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+import { isIPv4, isIPv6 } from "./ip.ts";
+import { assertEquals } from "@std/assert";
+
+Deno.test("isIPv4() returns boolean for list of IPv4s", () => {
```

Comment:
```suggestion
Deno.test("isIPv4()", () => {
```
This should suffice. Ditto for the other test.